queue/queue.go

```diff
@@ -27,6 +27,9 @@ type Store struct {
 func (db *Store) Next(ctx context.Context, resp mdm.Response) (*Command, error) {
 	dc, err := db.DeviceCommand(resp.UDID)
 	if err != nil {
+		if isNotFound(err) {
```

Comment:
I see your response now. I hadn't refreshed my page this morning so didn't see your reply. :)